src/libponyc/expr/ffi.c

```diff
@@ -62,10 +62,6 @@ static ast_result_t declared_ffi(pass_opt_t* opt, ast_t* call, ast_t* decl)
     {
       errorframe_t frame = NULL;
       ast_error_frame(&frame, arg, "argument not a subtype of parameter");
-      ast_error_frame(&frame, param, "parameter type: %s",
-        ast_print_type(p_type));
-      ast_error_frame(&frame, arg, "argument type: %s",
-        ast_print_type(a_type));
```

Comment:
It used to :) But now, the error information generated by `is_subtype` contains the same information, plus more detail.